Documentation/SpiderBasic/English/WebSocket.txt

```diff
@@ -148,9 +148,9 @@ WebSocket server implementation: @InternetLink "https://github.com/Dadido3/WebSo
   OpenWindow(0, 100, 100, 220, 50, "WebSocket test")
   ButtonGadget(0, 10, 10, 200, 30, "Send string !")
 
-  ; Connect to a free online websocket which send back every command
+  ; Connect to a free online websocket which sends back every command
   If OpenWebSocket(2, "wss://ws.postman-echo.com/raw") 
-    Debug "Trying to open the websocket"
+    Debug "Trying to open the web socket"
```

Comment:
Between this line and two lines above, you should be consistent beteen `websocket` and `web socket`. By the way there also other places where `WebSocket` and  `WebSocket` are used. But that an inconsistency in the original document, not your fix.